### PR TITLE
37 - Rearrange styles bootstrapping

### PR DESCRIPTION
**Pull request by matiasbeckerle**
Could you take a look guys? I need your review too @damianmuti. Thanks.

I needed to make several changes to achieve this PR. In one step it contains the "undo" for styles. In another step the setup for the MS UI framework using node-sass, compass, jeet. In the future we could add more grunt tasks and folder structure bootstrapping for UI guys.

Don't freak out if you see everything white & black because the idea is to jump up as soon as possible creating the properly UI instead of cleaning up frameworks that possible will not being used like Bootstrap.

Recommendations are welcome!

### CODE REVIEW

**Comment by damianmuti** [comment]
What an amazing job, Matt! Really glad to see all this work done. Kudos, buddy! :+1:

**Comment by matiasbeckerle** [comment]
Thanks to you @damianmuti !!!

**Comment by matiasbeckerle** [comment]
Could you take another look @damianmuti, @mravinale? Thanks!

**Comment by damianmuti** [comment]
Just checked. Everythings look fine.

File: README.md [hunk 1]
```diff
@@ -28,4 +28,16 @@ Run ```grunt dev``` to start the app in development mode with client-server live
 
 ### Testing
 
-Run ```grunt test``` to start the karma test runner.
\ No newline at end of file
+TODO: we need to improve testing configuration and related tasks.
```
Comment (mravinale) [inline README.md]: please move this to a new section called known errors in the wiki: https://github.com/MakingSense/mean-seed/wiki and place a link to this wiki on the bottom

Comment (matiasbeckerle) [inline README.md]: Good idea! Done.

**Comment by mravinale** [comment]
thank you very much for your dedication and work Kudos! :+1:

**Comment by matiasbeckerle** [comment]
Thanks guys!

**Pull request merged**